poke.io.js

```diff
@@ -63,12 +62,6 @@ function getNeighbors(lat, lng) {
 function Pokeio() {
   var self = this;
   self.events = new EventEmitter();
-  self.j = request.jar();
```

Comment:
This does have the drawback of making it so the `jar` and `request` objects in this `Pokeio` object can not be customized by external code. I'm not sure if this is an issue or not, nor do I have a specific use-case for this, but I thought I would point it out.